site/pages/docs/server-en.hbs

```diff
@@ -4,7 +4,7 @@
 	"language": "en",
 	"description": "Instructions for configuring the server.",
 	"altLangPrefix": "server",
-	"dateModified": "2014-10-15"
+	"dateModified": "2023-06-05"
 }
 ---
 
```

Comment:
I think we could remove this page completely as it seems to only discuss about old IE issue.

Reply:
That page's scope goes slightly beyond IE, but I agree that it's irrelevant at this point:

* [IIS 6.0-7.x are no longer supported.](https://learn.microsoft.com/en-us/lifecycle/products/internet-information-services-iis)... wouldn't surprise me if newer versions have SVG as a built-in MIME type.
* WET/GCWeb's page templates stopped using IE conditional comments long ago
* IE11 never supported the ``X-UA-Compatible`` flag
* IE compatibility mode is no longer a thing in Edge Chromium
* Edge Chromium's "IE mode" (IE11 Trident mode) can't be used for debugging... Edge's devtools are disconnected in that mode, making it impossible to view IE11's console if anything breaks

Reply:
Just scrapped this page and links that went towards it.